server/events/runtime/runtime.go

```diff
@@ -67,6 +73,15 @@ func GetPlanFilename(workspace string, projName string) string {
 	return fmt.Sprintf("%s-%s.tfplan", projName, workspace)
 }
 
+// isRemotePlan returns true if planContents are from a plan that was generated
+// using TFE remote operations.
+func IsRemotePlan(planContents []byte) bool {
+	// We add a header to plans generated by the remote backend so we can
+	// detect that they're remote in the apply phase.
+	remoteOpsHeaderBytes := []byte(remoteOpsHeader)
```

Comment:
This is tripping me up, where is `remoteOpsHeader` declared?

Reply:
https://github.com/lyft/atlantis/blob/master/server/events/runtime/plan_step_runner.go#L325:L326

Reply:
oh, weird. Searching didn't return anything.